src/plat/imx6/overlay-nitrogen6sx.dts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020, Hensoldt Cyber
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Do you prefer this to be under GPL or is BSD acceptable here also?

Reply:
It'd be nice to avoid another BSD variant, i.e. BSD-2-Clause instead of BSD-3-Clause would be good.

Dual-licensed would be optimal for people who worry about having to think about different licenses (I don't see any dependencies on GPL code here, so BSD-2-Clause would be fine, but people worry about all kinds of things and if it's easy to do, why not).

Reply:
There is https://github.com/seL4/seL4/blob/master/LICENSES/BSD-3-Clause.txt, so using 'BSD-3-Clause' should be fully covered already. Unless there is a plan to remove this?